### ISSUE TITLE
Add All Command

### ISSUE THREAD

**Issue by eduardoarandah** [oleg-shilo/sublime-favorites#8]
Oleg! 
it's me again

An "Add All Command" would be very very good. 

Let's say you have to spend the next couple of hours working on some screen. 
The workflow would be.. 
openning the controller, the view, the scss file, the model, the bootstrap _variables, bla bla.. 

then "Add All to Favorites" and start working... 

and then... the next day remove those favorites, add the new ones and start again.. 

what do you think?

**Comment by oleg-shilo** [comment]
Agree. It's a simple to implement and safe to release feature. Will look at it.
